### DISCUSSION TITLE
We need to ensure users from one tenant are not able to access data from different tenant

### DISCUSSION THREAD

**Discussion by jhutar** [Hyperfoil/Horreum#424] [General]
**Task.**

We need to ensure users from one tenant are not able to access data from different tenant

**Describe the solution you'd like**

Somebody with a pen-testing mindset to test the application and ensure this is OK.

**Additional information**

Reporting this because it was raised during review of https://github.com/redhat-appstudio/book/pull/66

**Comment by jesperpedersen** [comment]
Horreum is based on access control which uses PostgreSQL Row Level Security.

If you have questions about the project use our general forum,

 https://github.com/Hyperfoil/Horreum/discussions/categories/general

please.

In case you have found some improvements to the model, UI, default values use our ideas forum,

 https://github.com/Hyperfoil/Horreum/discussions/categories/ideas

please.

Thanks.
